src/test/java/uk/gov/hmcts/reform/preapi/controller/MediaServiceControllerTest.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.hmcts.reform.preapi.controller;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.reform.preapi.controllers.MediaServiceController;
+import uk.gov.hmcts.reform.preapi.media.AzureMediaService;
+import uk.gov.hmcts.reform.preapi.security.service.UserAuthenticationService;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@WebMvcTest(MediaServiceController.class)
+@AutoConfigureMockMvc(addFilters = false)
+public class MediaServiceControllerTest {
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private AzureMediaService mediaService;
+
+    @MockBean
+    private UserAuthenticationService userAuthenticationService;
+
+    @DisplayName("Should return 200 when successfully connected to media service")
+    @Test
+    void getMediaSuccess() throws Exception {
+        when(mediaService.getAssets()).thenReturn(List.of());
+
+        var response = mockMvc.perform(get("/media-service/health"))
+            .andExpect(status().isOk())
+            .andReturn().getResponse();
+
+        assertThat(response.getContentAsString()).isEqualTo("successfully connected to media service");
+    }
+
+
+    //    @DisplayName("Should return 500 when cannot connect to media service")
```

Comment:
Does this need outright deleting or is it useful for later?

Reply:
No I have just commented this out because I am currently using this error message to help me test the helm issue I'm trying to fix